cf_xarray/accessor.py

```diff
@@ -860,7 +871,12 @@ def _get_all_cell_measures(self):
 
         # get all_cell_measures only once
         if not self._all_cell_measures:
-            self._all_cell_measures = set(_CELL_MEASURES + tuple(self.cell_measures))
+            self._all_cell_measures = set(_CELL_MEASURES)
+        try:
+            self._all_cell_measures |= set(self.cell_measures)
+        except ValueError:
+            # ignore bad cell_measures attribute
```

Comment:
Add a warning here? E.g., we could use the same message of the error.

Reply:
how about now? Raise an error with `.cf.cell_measures` but warn in `_getitem`. I think that's better

Reply:
Good idea!